cmake/opendds_testing.cmake

```
@@ -20,7 +20,7 @@ function(opendds_add_test)
   list(APPEND arg_COMMAND ${arg_ARGS})
   add_test(NAME "${test_name}" ${arg_UNPARSED_ARGUMENTS} COMMAND ${arg_COMMAND})
 
-  set(env "ACE_ROOT=${ACE_ROOT}" "TAO_ROOT=${TAO_ROOT}")
+  set(env "ACE_ROOT=${ACE_ROOT}" "TAO_ROOT=${TAO_ROOT}" "DDS_ROOT=${DDS_ROOT}")
```

```suggestion
  set(env "ACE_ROOT=${ACE_ROOT}" "TAO_ROOT=${TAO_ROOT}")
```

I'm explicitly not setting this because it shouldn't be necessary and I'm trying to eventually get rid of it.